### PR TITLE
Robynpy pareto cleanup

### PR DESCRIPTION
**Pull request by dhavalpatel624624**
## Project Robyn
Code clean up primarily for Pareto:
- Moved ParetoResult into entities folder
- Fixed Pareto_optimizer for 100% accuracy 
- Made min_max_norm not private so that it can be called from Clustering also

Running unit tests of test_pareto_utils
image

### CODE REVIEW

**Comment by sumane81** [comment]
You want to merge this to robynpy_clustering or robynpy_release?

File: python/src/robyn/modeling/pareto/pareto_optimizer.py [hunk 1]
```diff
@@ -22,34 +23,6 @@
...
-    x_decomp_agg: pd.DataFrame
-    result_calibration: Optional[pd.DataFrame]
-    media_vec_collect: pd.DataFrame
-    x_decomp_vec_collect: pd.DataFrame
-    plot_data_collect: Dict[str, pd.DataFrame]
-    df_caov_pct_all: pd.DataFrame
-
-
 @dataclass
 class ParetoData:
```
Comment (sumane81) [inline python/src/robyn/modeling/pareto/pareto_optimizer.py:27]: Can this dataclass also moved to entities?

**Review by sumane81 — CHANGES_REQUESTED** [review]

File: python/src/robyn/modeling/pareto/pareto_optimizer.py (hunk 1 above)
Comment (dhavalpatel624624) [reply python/src/robyn/modeling/pareto/pareto_optimizer.py:27]: It is only used as an object to return data from within pareto_optimizer. Its not really meant to be used outside of pareto_optimizer.

File: python/tests/modeling_pareto/test_pareto_utils.py
[the remark this answers is not in the document]
Comment (dhavalpatel624624) [reply python/tests/modeling_pareto/test_pareto_utils.py:1]: Yes I'll do that

**Comment by dhavalpatel624624** [comment]
This is temporarily showing its merged into robynpy_clustering only to limit what you see in the PR because this will be getting merged after the other PR. I'll change it to robynpy_release after I get an approval on this one and other one is merged.

**Review by sumane81 — APPROVED** [review]

**Review by sumane81 — APPROVED** [review]
Looks good to me.

**Pull request merged**